backend/sirius-web-collaborative-diagrams-api/src/main/java/org/eclipse/sirius/web/collaborative/diagrams/api/dto/DeleteFromDiagramInput.java

```diff
@@ -49,16 +53,21 @@ public UUID getRepresentationId() {
         return this.representationId;
     }
 
-    @GraphQLID
     @GraphQLField
     @GraphQLNonNull
-    public String getDiagramElementId() {
-        return this.diagramElementId;
+    public List<@GraphQLNonNull String> getNodeIds() {
+        return this.nodeIds;
+    }
+
+    @GraphQLField
+    @GraphQLNonNull
+    public List<@GraphQLNonNull String> getEdgeIds() {
```

Comment:
This should be `List<@GraphQLNonNull @GraphQLID String>` or something like that

Reply:
This comment is interesting.
You say that an edgeId is an UUID but the getIdProvider return a String.
If I see the EdgeMappingConverter, the idProvider give a string ``` sourceId + " --> " + targetId + " - " + count```
(It's not a disguised UUID)

This @GraphQLID implies :
* Edge#id
* EdgeDescription#idProvider
* EdgeElementProps
* IDiagramService#findEdgeById

This update should be done in a dedicated enhancement.